### ISSUE TITLE
Add captions to Color video in Carbon documentation

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon-website#2161]
Accessibility failure on Carbon site due to no captions on video. The example is on https://www.carbondesignsystem.com/guidelines/color/overview There may be others.

The player itself lacks a caption button, so there are really 2 key things to resolve:
1. Ensure all videos used in Carbon are displayed in a player that has captioning controls
2. Ensure captions for this video are generated and embedded
